game-core/src/main/java/games/strategy/triplea/delegate/data/MoveDescription.java

```diff
@@ -50,11 +46,11 @@ public String toString() {
     return "Move message route:" + route + " units:" + getUnits();
   }
 
-  public Collection<Unit> getTransportsThatCanBeLoaded() {
-    if (transportsThatCanBeLoaded == null) {
-      return Collections.emptyList();
+  public Map<Unit, Unit> getUnitsToTransports() {
+    if (unitsToTransports == null) {
+      return Map.of();
     }
-    return transportsThatCanBeLoaded;
+    return unitsToTransports;
```

Comment:
What about preventing `unitsToTransports` to be null in the first place?
(This check could be done in the constructor already)